test/e2e/framework/util.go

```diff
@@ -439,14 +439,16 @@ func countEndpointsNum(e *v1.Endpoints) int {
 
 // restclientConfig returns a config holds the information needed to build connection to kubernetes clusters.
 func restclientConfig(kubeContext string) (*clientcmdapi.Config, error) {
-	Logf(">>> kubeConfig: %s", TestContext.KubeConfig)
+	//TODO @perithompson work out if we need this logging now
```

Comment:
why all these changes to the e2e framework?

Reply:
we can back these out , long story :) but will file a separate issue